src/Microsoft.Identity.Client/UI/CoreUIParent.cs

```diff
@@ -32,10 +32,13 @@
 #if iOS
 using UIKit;
 #endif
+#if MAC
+using AppKit;
+#endif
 
 namespace Microsoft.Identity.Client.UI
 {
-    internal class CoreUIParent
+    internal class CoreUIParent //TODO bogavril: break this class up into platform specific classes or merge with UIParent (probably merge!)
```

Comment:
I've had the same thought as your comment.  let's collaborate to clean this up.  For public API (UiParent) we also need to make some changes so we should chat more here.